lib/src/components/NxTransferListHalf/NxTransferListHalf.tsx

```diff
@@ -68,13 +68,15 @@ function TransferListItem<T extends string | number = string>(props: TransferLis
             <NxButton type="button"
                       variant="icon-only"
                       title={moveUpButtonTitle}
+                      aria-label={moveUpButtonTitle || `${moveUpButtonTitle} (disabled)`}
```

Comment:
This shouldn't be necessary, the `title` should be the accessible name

Reply:
> This shouldn't be necessary, the `title` should be the accessible name

Oh right. I think this is a better approach: 3c8882aff25a511c08e893521cf0a9bf8adfd6ad